fe/fe-core/src/main/java/com/starrocks/mysql/privilege/Auth.java

```diff
@@ -390,6 +394,9 @@ public boolean checkTblPriv(ConnectContext ctx, String qualifiedDb, String tbl,
     }
 
     public boolean checkTblPriv(UserIdentity currentUser, String db, String tbl, PrivPredicate wanted) {
```

Comment:
CheckTblWithDb should change too?

Reply:
checkTblwithdb is an private method, only be used in checkTblPriv, and before that we have made db->fullnamedb